configuration/concepts/lab_tests.csv

```diff
@@ -55,23 +55,30 @@ d04d3d38-a009-4e7f-a8a0-49e4be99cfbd,,,Blood in urine,Sang,Blood in urine,Sang,,
 61c0f1d9-6440-4a22-9c55-b10603245254,,,Bilirubin,Bilirubine,Bilirubin,Bilirubine,,,LabTest,Text,,,,,,,,,,,
 bb655797-22dc-4d21-9806-88fbc62cd601,,CIEL:1369,Urobilinogen,Urobilinogène,Urobilinogen,Urobilinogène,,,LabTest,Text,,,,,,,,,,,
 a360c76e-b365-4854-97f9-1703ec29cb48,,,Urine Protein,Urine Protéines,Urine Protein,Urine Protéines,,,LabTest,Text,,,,,,,,,,,
-e5dc4fb9-5a3c-4956-9a39-66aaaf64adcc,,,Leucocytes,Leucocytes,Leucocytes,Leucocytes,,,LabTest,Text,,,,,,,,,,,
-e3ae7371-8a91-413e-8f99-93bc8f45a07a,,,White Blood Cells,Globules blancs,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
-668b296c-a1fc-4d2f-bfad-1997704a4762,,,Red Blood Cells,Globules rouges,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
-168f4adf-4beb-49d2-8070-b2776ccd2b23,,,Bacteria,Bactéries,Bacteria,Bactéries,,,LabTest,Text,,,,,,,,,,,
-1d832608-72a0-4e18-af6c-5d3bc2c18caf,,,Epithelial Cells,Cellules épithéliales,Epithelial Cells,Cellules épithéliales,,,LabTest,Text,,,,,,,,,,,
-d2f224b3-7604-4ae0-a7ae-93ea92ce7106,,,Budding Yeasts,Levures bourgeonnates,Budding Yeasts,Levures bourgeonnates,,,LabTest,Text,,,,,,,,,,,
-8ac2c2c6-5333-43ac-b2ee-25077160b443,,,Simple Yeasts,Levures simples,Simple Yeasts,Levures simples,,,LabTest,Text,,,,,,,,,,,
-93cbd183-498b-4dd0-8002-b0d49d00709c,,,Mycelium,Miscelium,Mycelium,Miscelium,,,LabTest,Text,,,,,,,,,,,
+e3ae7371-8a91-413e-8f99-93bc8f45a07a,,,Urine White Blood Cells,Globules blancs urinaires,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
+eceed5e9-c513-4ff0-b8a2-900f53599c41,,,Swab White Blood Cells,Écouvillon de globules blancs,White Blood Cells,Globules blancs,,,LabTest,Text,,,,,,,,,,,
+668b296c-a1fc-4d2f-bfad-1997704a4762,,,Urine Red Blood Cells,Globules rouges urinaires,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
+d4708141-0562-49c4-aabe-b03a2dbf7c2b,,,Swab Red Blood Cells,Écouvillon de globules rouges,Red Blood Cells,Globules rouges,,,LabTest,Text,,,,,,,,,,,
+168f4adf-4beb-49d2-8070-b2776ccd2b23,,,Urine Bacteria,Bactéries urinaires,Bacteria,Bactéries,,,LabTest,Text,,,,,,,,,,,
+02a4cfd9-196c-4ce7-90a8-c67f52c8fa7e,,,Swab Bacteria,Bactéries sur écouvillon,Bacteria,Bactéries,,,LabTest,Text,,,,,,,,,,,

```

Comment:
Not sure if it should be "Prélèvement de bactéries" or "Bactéries sur prélèvement". Depends on what Swab Bacteria is really.

Reply:
Done